.planning/phases/01-tooling-baseline-python-matrix/01-PATTERNS.md

```
@@ -0,0 +1,790 @@
+```
+(line 45)
+
+```markdown
+The latest stable version of Python 3 is the default, and target
+version accessible as the `cement` container within Docker Compose.
+For testing against alternative versions of python, additional
+containers are created (ex: `cement-py39`, `cement-py310`, etc).
+```
+(line 136)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)